playground/backend/pkg/environment/environment_service.go

```diff
@@ -0,0 +1,44 @@
+package environment
+
+import (
+	"os"
+	"strconv"
+)
+
+const (
+	serverIpKey   = "SERVER_IP"
+	serverPortKey = "SERVER_PORT"
+	defaultIp     = "localhost"
+	defaultPort   = 8080
+)
+
+type Service struct {
+	ServerEnvs ServerEnvs
+	LogWriters LogWriters
+}
+
+func NewService(writers *LogWriters) *Service {
+	if writers == nil {
+		writers = NewLogWriters(os.Stdout, os.Stdout, os.Stderr)
+	}
+	svc := Service{LogWriters: *writers}
+	svc.initServerEnvsFromOsEnvs()
+
+	return &svc
+}
+
+func (svc *Service) initServerEnvsFromOsEnvs() {
+	ip := defaultIp
+	port := defaultPort
+	if value, present := os.LookupEnv(serverIpKey); present == true {
+		ip = value
+	}
+
+	if value, present := os.LookupEnv(serverPortKey); present == true {
+		if converted, err := strconv.Atoi(value); err != nil {
+			port = converted
```

Comment:
Should we somehow log that we couldn't convert the provided port for the server?